apps/files_sharing/js/app.js

```diff
@@ -303,7 +303,7 @@ OCA.Sharing.App = {
 				if (context.$file.attr('data-remote-id')) {
 					shareBase = 'remote_shares/pending'
 				}
-				$.post(OC.linkToOCS('apps/files_sharing/api/v1/shares/pending', 2) + shareId)
+				$.post(OC.linkToOCS('apps/files_sharing/api/v1/remote_shares/pending', 2) + shareId)
```

Comment:
the question for me is if internal shares will still work with this fix or if we need to add a switch or condition based on internal or federated sharing...

Reply:
Test it?

Reply:
maybe on monday ;)

Reply:
Hmmm... which internal shares trigger an accept/reject action? As far as I have experienced it, only external shares ask for confirmation, or did I overlook any functionality? Depending on the answer to that topic, there is either the variable shareBase to be used again, or some dead code to be cleaned up...

Reply:
After having a deeper look at the logic I think this should work:
```suggestion
				$.post(OC.linkToOCS('apps/files_sharing/api/v1/' + shareBase, 2) + shareId)
```
Please report back if it works! Thank you! :)

Reply:
> which internal shares trigger an accept/reject action?

I guess you are looking for this:
https://github.com/nextcloud/server/blob/b8b4d247b4382d7c51323976a3a17e1416dcfe0a/config/config.sample.php#L1562-L1571

Reply:
OK, clear and understood (did not know about that)

Reply:
Quick question, before messing things up: 
Should I initiate another force-push to the PR or can I accept the proposal in here ("commit suggestion")?

Reply:
Honestly both is fine but please don't forget the sign-off :)

Force-pushing is probably a bit nicer though :)

Reply:
OK, force-pushed correction - my own tests for remote share and local share (I temporarily have set the sharing.force_share_accept option) now also works. So good to go from my side, but please also test on your end.